pom.xml

```diff
@@ -97,7 +97,7 @@
     <src.output>${basedir}/target</src.output>
     <imageio.ext.version>1.2.0</imageio.ext.version>
     <jts.version>1.16.0</jts.version>
-    <jaiext.version>1.1.4</jaiext.version>
+    <jaiext.version>1.1-SNAPSHOT</jaiext.version>
```

Comment:
I think you set it to SNAPSHOT for development and testing. should it be updated to 1.1.next once Jai-ext get released?

Reply:
That's why I said "Depends on geosolutions-it/jai-ext#227 **and a jai-ext release**" :-D